sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/implementation/ImportJobsImpl.java

```diff
@@ -0,0 +1,689 @@
+import com.azure.core.annotation.ServiceInterface;
+import com.azure.core.annotation.ServiceMethod;
+import com.azure.core.annotation.UnexpectedResponseExceptionType;
+import com.azure.core.http.rest.PagedFlux;
+import com.azure.core.http.rest.PagedResponse;
+import com.azure.core.http.rest.PagedResponseBase;
+import com.azure.core.http.rest.Response;
+import com.azure.core.http.rest.RestProxy;
+import com.azure.core.util.Context;
+import com.azure.core.util.FluxUtil;
+import com.azure.digitaltwins.core.implementation.models.ErrorResponseException;
+import com.azure.digitaltwins.core.implementation.models.ImportJob;
+import com.azure.digitaltwins.core.implementation.models.ImportJobCollection;
+import com.azure.digitaltwins.core.implementation.models.ImportJobsAddOptions;
+import com.azure.digitaltwins.core.implementation.models.ImportJobsCancelOptions;
+import com.azure.digitaltwins.core.implementation.models.ImportJobsDeleteOptions;
+import com.azure.digitaltwins.core.implementation.models.ImportJobsGetByIdOptions;
+import com.azure.digitaltwins.core.implementation.models.ImportJobsListOptions;
+import reactor.core.publisher.Mono;
+
+/**
+ * An instance of this class provides access to all the operations defined in ImportJobs.
+ */
+public final class ImportJobsImpl {
+    /**
+     * The proxy service used to perform REST calls.
+     */
+    private final ImportJobsService service;
+
+    /**
+     * The service client containing this operation class.
+     */
+    private final AzureDigitalTwinsAPIImpl client;
+
+    /**
+     * Initializes an instance of ImportJobsImpl.
+     *
+     * @param client the instance of the service client containing this operation class.
+     */
+    ImportJobsImpl(AzureDigitalTwinsAPIImpl client) {
+        this.service
+            = RestProxy.create(ImportJobsService.class, client.getHttpPipeline(), client.getSerializerAdapter());
+        this.client = client;
+    }
+
+    /**
+     * The interface defining all the services for AzureDigitalTwinsAPIImportJobs to be used by the proxy service to
+     * perform REST calls.
+     */
+    @Host("{$host}")
+    @ServiceInterface(name = "AzureDigitalTwinsAPI")
+    public interface ImportJobsService {
+        /**
+         * Lists all import jobs.
+         *
+         * @param host The host URL.
+         * @param traceparent The traceparent header.
+         * @param tracestate The tracestate header.
+         * @param maxItemsPerPage The maximum number of items per page.
+         * @param apiVersion The API version.
+         * @param accept The accept header.
+         * @param context The context to associate with this operation.
+         * @return A Mono that emits the response containing the list of import jobs.
+         */
+        @Get("/jobs/imports")
+        @ExpectedResponses({ 200 })
+        @UnexpectedResponseExceptionType(ErrorResponseException.class)
+        Mono<Response<ImportJobCollection>> list(@HostParam("$host") String host,
```

Comment:
These are new APIs that are not added to the public client. With the new service version, these new APIs should be exposed in DigitalTwinsClient and DigitalTwinsAsyncClient.